pom.xml

```diff
@@ -179,7 +179,6 @@
 
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <server.jvm.args>${jvm.args.other} ${jvm.args.memory} ${jvm.args.debug} ${jvm.args.modular}</server.jvm.args>
-    <skip.maven-deploy-plugin>false</skip.maven-deploy-plugin>
```

Comment:
The team is discussing this change internally. I don't think it is worth to copy&paste/summarise the internal conversation here. As soon as we have an outcome, we will update the community here